addons/mail/static/tests/helpers/test_utils.js

```diff
@@ -188,7 +188,12 @@ const VIEW_TYPE_TO_ARCH = {
                      * @returns {integer[]|integer} array of ids if more than one value was passed,
                      * id of created record otherwise.
                      */
-                    create(values = {}) {
+                    create(values) {
+                        if(!values) {
+                            // Support for create() without parameter is dropped.
+                            // Use create({}) instead.
```

Comment:
This comment is not necessary, it explains the diff, but not the code itself.
It's fine to just do the return, the goal is to simulate python and it's what is done in python.